<?xml version="1.0" encoding="UTF-8"?>
    <source package="javax.management">
        <class name="JMRuntimeException" line="2">
            <extends class="RuntimeException"/>
            <javadoc line="2">
                Runtime exceptions emitted by JMX implementations.                
                <since>
                    1.5                    
                </since>
            </javadoc>
            <declaration name="serialVersionUID" type="long" line="7"/>
            <javadoc line="8">
                Default constructor.                
            </javadoc>
            <method name="JMRuntimeException" type="constructor" line="11"/>
            <javadoc line="14">
                Constructor that allows a specific error message to be specified.                
                <param>
                    message the detail message.                    
                </param>
            </javadoc>
            <method name="JMRuntimeException" type="constructor" line="18">
                <params>
                    <param name="message" type="String"/>
                </params>
            </method>
            <javadoc line="21">
                Constructor with a nested exception.  This constructor is
                  package-private because it arrived too late for the JMX 1.2
                  specification.  A later version may make it public.                
            </javadoc>
            <method name="JMRuntimeException" type="constructor" line="26">
                <params>
                    <param name="message" type="String"/>
                    <param name="cause" type="Throwable"/>
                </params>
            </method>
        </class>
    </source>